javascript/javascript1/week1/Housy-Pricy.js

```diff
@@ -0,0 +1,29 @@
+var width = 8;
+var depth = 10;
+var height = 10;
```

Comment:
`let` not `var`. In general, try to use `const` always and if you will change the value after use `let`